### ISSUE TITLE
Text on welcome screens cut off - Samsung S20 FE.

### ISSUE THREAD

**Issue by Pharste** [StatCan/StatcanPublicationUAT#298]
Hi, 

A user submitted these screenshots, showing cut-off text on her Samsung S20 FE device. I've asked for what software the device is running, and will update the ticket when I hear back. 
![Samsung S20 FE 3](https://user-images.githubusercontent.com/89025880/152048014-bc1e8419-4574-48ae-81df-847240ff1ae5.jpg)
![Samsung S20 FE 1](https://user-images.githubusercontent.com/89025880/152048080-3e7fc00b-9e31-4a90-b42f-4ae185f9fdb4.jpg)

**Comment by Pharste** [comment]
From the user - "showing last update was installed on November 22, 2021.  Is says it is Android version 11.  And One UI version 3.1 is in the list just above the Android version"

**Comment by goodyke** [comment]
Hi @Rajat-Sen,
I will remove myself as an assignee as I have no method to test an Android device. Please look into this concern when you have time. Thanks!

**Comment by AJBouzidi** [comment]
@Pharste  could you add device information?

**Comment by AJBouzidi** [comment]
OS: Android version 11.
Device: Samsung S20 FE

**Comment by payeisa** [comment]
The issue with this ticket seems to be based on the user's phone settings. I accessed the app on the Samsung S20, and screens were displaying fine:
![image](https://user-images.githubusercontent.com/89025704/167118244-fbb18fa8-5f92-453e-b82d-75d894a5bdad.png)
![image](https://user-images.githubusercontent.com/89025704/167118275-3c7d97a3-9e48-4366-bc4b-961a8afb0067.png)

However, if I increase the size from the phone's settings (Settings - Display - Font size and style), then I'm able to replicate the issue:
![image](https://user-images.githubusercontent.com/89025704/167118567-57252a89-89c1-4f39-afea-401a83672fee.png)

Is there anything we can do on the app side to address this?

Isabelle

**Comment by ForumG-SC** [comment]
> The issue with this ticket seems to be based on the user's phone settings. I accessed the app on the Samsung S20, and screens were displaying fine: ![image](https://user-images.githubusercontent.com/89025704/167118244-fbb18fa8-5f92-453e-b82d-75d894a5bdad.png) ![image](https://user-images.githubusercontent.com/89025704/167118275-3c7d97a3-9e48-4366-bc4b-961a8afb0067.png)
> 
> However, if I increase the size from the phone's settings (Settings - Display - Font size and style), then I'm able to replicate the issue: ![image](https://user-images.githubusercontent.com/89025704/167118567-57252a89-89c1-4f39-afea-401a83672fee.png)
> 
> Is there anything we can do on the app side to address this?
> 
> Isabelle

Looking into this issue, fix will be applied soon to test it out.

**Comment by payeisa** [comment]
@zhuelle Priority added.

**Comment by giftydeep** [comment]
@Jiandong-Guo The tab icons are still cutting off. Here is the screenshot attached below. Please take a look.
Device Info:
Model : A10E
Screen Size: 5.83
Android Version : 11
![Screenshot_20220705-093449_Expo Go](https://user-images.githubusercontent.com/8207028/177380314-dd3422c4-7c90-4436-8aab-d20e4bd83aef.jpg)

**Comment by giftydeep** [comment]
waiting on wifi to discuss this ticket in person with Jiandong.

**Comment by Jiandong-Guo** [comment]
Actually these are 2 issues:
1)The welcome screen is a simple style problem and nothing related to 338, and nothing related to tab, because when it shows the tab is not created yet.   
2)The Tab control cut-off issue, I think it doesn't exist anymore since we changed the tab control long time ago, please check it

**Comment by Jiandong-Guo** [comment]
@giftydeep 
This is the screen shot from android 8-Fold_out emulator:
![image](https://user-images.githubusercontent.com/58485300/187462551-f5b35b8c-4519-4e86-a699-d046b197b040.png)

**Comment by giftydeep** [comment]
Since @Jiandong-Guo is using Samsung Fold for testing, which has very wide screen and completely different aspect ratio than other Samsung Phone. Also the Height and Width is hard coded for some of the screen that's why it is not very responsive when the screen sizes change. At this moment this issue has been fixed on all other models except Fold. I will be creating new ticket for fixing Height and Width issue.

**Comment by levegab** [comment]
This issue has been resolved. 
![Screenshot_20220914-152604_StatsCAN](https://user-images.githubusercontent.com/106617357/190233749-25147517-9552-4cda-9954-6b8cf5bb96af.jpg)
